app/scripts/modules/netflix/fastProperties/dataNav/fastPropertyFilterSearch.less

```diff
@@ -7,12 +7,14 @@
 }
 
 .fast-property-filter-search {
+  position: relative;
   margin-left: 10px;
   margin-top: 6px;
   .dropdown-menu {
+    position: absolute;
     min-width: 206px;
-    top: 38px;
-    left:560px;
+    top: 35px;
+    left:65px;
```

Comment:
Yes, I also switched the positioning to be absolute within a relative container.